### PR TITLE
SvelteKit docs and examples - remove `throw`

### PR DESCRIPTION
**Pull request by kirso**
## What kind of change does this PR introduce?
Changes to documentation and examples section

## What is the current behavior?
Current documentation and examples still use SvelteKit 1.0. and `throw` for errors

## What is the new behavior?

In SvelteKit 2.0 removes the need to use the `throw` for errors / redirects.

Documentation link: https://kit.svelte.dev/docs/errors

## Additional context

Not a critical change, just a short cleanup

### CODE REVIEW

**Review by charislam — APPROVED** [review]

**Review by charislam — APPROVED** [review]

**Comment by kirso** [comment]
Not sure why its not merging, seems like there are still some changes that have happened before the approval.

**Comment by charislam** [comment]
Just needed to redeploy staging post-merge, I think. All's good now, thank you!

**Pull request merged**
